src/js/main.js

```diff
@@ -442,6 +442,9 @@ Site.Portraits = {
 
     // Bind arrange complete grid event
     _this.$grid.on('arrangeComplete',_this.handleArrangeComplete.bind(_this));
+
+    // Bind scroll to top on portrait click
+    $('.portrait a').on('click', Site.Player.scrollIn());
```

Comment:
The callback should not have `()` otherwise is not getting passed as a callback but executed at runtime.

wrong:
```
$('.portrait a').on('click', Site.Player.scrollIn());
```
right:
```
$('.portrait a').on('click', Site.Player.scrollIn);`
```

Reply:
Also Idk how, i really don't understand, but this breaks the player too
```
main.js?ver=4.8.4:302 Uncaught TypeError: Cannot read property 'loadVideoById' of undefined
    at Object.playVideo (main.js?ver=4.8.4:302)
```

commenting it in fixes or brakes the player, which is really weird

Reply:
ok i fix